### PR TITLE
#135 - Use uima style offsets

### PR DESCRIPTION
**Pull request by jcklie**
- Create an index to map between UIMA and cassis offsets

Test it e.g. via:

```
from cassis import *

TOKEN = "de.tudarmstadt.ukp.dkpro.core.api.segmentation.type.Token"

with open("TypeSystem.xml", "rb") as f:
    typesystem = load_typesystem(f)

with open("smileys.xmi", "rb") as f:
    cas = load_cas_from_xmi(f, typesystem=typesystem)

for token in cas.select(TOKEN):
    print(token.get_covered_text())
```

Or import a document with that text into INCEpTION.

Before:

```
Hello
😊,
 
y 
ame 
s 
ohn.
```

After:

```
Hello
😊
,
my
name
is
John
.
```

### CODE REVIEW

File: tests/test_files/xmi/cas_with_nonindexed_fs.xmi [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
   <cas:NULL xmi:id="0"/>
-  <type2:TagsetDescription xmi:id="2" sofa="1" begin="0" end="684" tags="3 4 5"/>
```
Comment (reckart) [inline tests/test_files/xmi/cas_with_nonindexed_fs.xmi:3]: Any idea why the end offset changed here so drastically?

File: tests/test_xmi.py [hunk 2]
```diff
@@ -66,9 +67,9 @@ def test_views_are_parsed(small_xmi, small_typesystem_xml):
     <xmi:XMI xmlns:tcas="http:///uima/tcas.ecore" xmlns:xmi="http://www.omg.org/XMI" xmlns:cas="http:///uima/cas.ecore"
              xmlns:cassis="http:///cassis.ecore" xmi:version="2.0">
         <cas:NULL xmi:id="0"/>
-        <tcas:DocumentAnnotation xmi:id="8" sofa="1" begin="0" end="47" language="x-unspecified"/>
```
Comment (reckart) [inline tests/test_xmi.py:69]: Again, I wonder why the offsets changed here and in the following lines...?

File: tests/test_files/xmi/cas_with_nonindexed_fs.xmi (hunk 1 above)
Comment (jcklie) [reply tests/test_files/xmi/cas_with_nonindexed_fs.xmi:3]: Wrong before, but we never checked them.

File: tests/test_xmi.py (hunk 2 above)
Comment (jcklie) [reply tests/test_xmi.py:69]: Wrong before, but we never checked them.

**Pull request merged**